### PR TITLE
e2e tests for edgesite

### LINKED ISSUE #624: Run e2e tests with edgesite implementation
**What would you like to be added**:
Run KubeEdge application deployment test for edgesite

**Why is this needed**:
The kubeedge e2e tests related to application deployment can ensure application deployment scenarios for edgesite. Just need to run edgesite binary instead of edgecontroller and edge_core.

### PR DESCRIPTION
**Pull request by pavan187**
**What type of PR is this?**
/kind test

**What this PR does / why we need it**:
E2E tests added for EdgeSite feature.

**Which issue(s) this PR fixes**:
Fixes #624

**Special notes for your reviewer**:

### CODE REVIEW

File: tests/e2e/constants/constants.go [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+const (
+	RunController = "cd ${GOPATH}/src/github.com/kubeedge/kubeedge/cloud/; sudo nohup ./edgecontroller > edgecontroller.log 2>&1 &"
+	RunEdgecore   = "cd ${GOPATH}/src/github.com/kubeedge/kubeedge/edge/; sudo nohup ./edge_core > edge_core.log 2>&1 &"
+	RunEdgeSite   = "cd ${GOPATH}/src/github.com/kubeedge/kubeedge/edgesite/; sudo nohup ./edgesite > edgesite.log 2>&1 &"
+
+	AppHandler        = "/api/v1/namespaces/default/pods"
+	NodeHandler       = "/api/v1/nodes"
+	DeploymentHandler = "/apis/apps/v1/namespaces/default/deployments"
+)
```
Comment (sids-b) [inline tests/e2e/constants/constants.go]: please add a blank line at the end of this file.
Applicable to similar cases below

Comment (pavan187) [reply tests/e2e/constants/constants.go]: done

File: tests/e2e/constants/constants.go [hunk 2]
```diff
@@ -0,0 +1,11 @@
+package constants
+
+const (
+	RunController = "cd ${GOPATH}/src/github.com/kubeedge/kubeedge/cloud/; sudo nohup ./edgecontroller > edgecontroller.log 2>&1 &"
```
Comment (rohitsardesai83) [inline tests/e2e/constants/constants.go]: it doesn't look good to have such commands defined as constants.
can we have a script with a function which can be executed via os.exec ?

Comment (rohitsardesai83) [reply tests/e2e/constants/constants.go]: @pavan187 , let's create an issue for this. similar comments for device management e2e test suite from Sujith.

Comment (pavan187) [reply tests/e2e/constants/constants.go]: > it doesn't look good to have such commands defined as constants.
> can we have a script with a function which can be executed via os.exec ?

OK sure, will change as suggested

**Pull request merged**